tools/gen_ort_dockerfile.py

```diff
@@ -183,6 +183,14 @@ def dockerfile_for_linux(output_file):
     df += '''
 WORKDIR /workspace/onnxruntime
 ARG COMMON_BUILD_ARGS="--config ${ONNXRUNTIME_BUILD_CONFIG} --skip_submodule_sync --parallel --build_shared_lib --build_dir /workspace/build --cmake_extra_defines CMAKE_CUDA_ARCHITECTURES='52;60;61;70;75;80;86' "
+'''
+
+    # Remove version info from libonnxruntime.so
+    # This makes it possible to replace ort binaries in released triton containers
+    # for experimentation, without having to build triton-ort backend.
+    df += '''
+RUN sed -i 's/VERS_%s//' tools/ci_build/gen_def.py &&  (sed -i 's/% VERSION_STRING//' tools/ci_build/gen_def.py) 
```

Comment:
Do you need to do something for Windows?

Reply:
No this is specific to linux

Reply:
Right now we are running into this issue for linux deployments and A/B testing. This specific solution is specific to linux... I havent tested if this is a problem for windows... This PR targets windows only